### PR TITLE
chore: Remove v2 branches from starters

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: packages/cli/medusa-cli/src/commands/new.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ const clone = async (hostInfo, rootPath, v2 = false, inputBranch) => {
       : []
 
   if (v2) {
-    branch = [`-b`, inputBranch || "feat/v2"]
+    branch = [`-b`, inputBranch || "master"]
```
Comment (shahednasser) [inline packages/cli/medusa-cli/src/commands/new.ts:190]: We don't need this option anyway so better to remove this.

**Review by shahednasser — APPROVED** [review]
LGTM just one suggestion

File: packages/cli/medusa-cli/src/commands/new.ts (hunk 1 above)
Comment (olivermrbl) [reply packages/cli/medusa-cli/src/commands/new.ts:190]: The `inputBranch`?

Comment (shahednasser) [reply packages/cli/medusa-cli/src/commands/new.ts:190]: The `v2` condition, we don't need a `--v2` option anymore

Comment (olivermrbl) [reply packages/cli/medusa-cli/src/commands/new.ts:190]: I noticed this is a more comprehensive change, so I'll save this cleanup for later.

**Pull request merged**